src/app/core/services/auth.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+
+  constructor() { }
+
+  login(user: string) {
+    localStorage.setItem('user', JSON.stringify({ login: user, token: Math.random().toString(36).substr(2) }));
```

Comment:
I would suggest you to create a separate service for storage. You can ready here about it: https://medium.com/fafnur/localstorage-sessionstorage-%D0%B4%D0%BB%D1%8F-angular-universal-a1393940a29f
And you can do simplified version of LocalStorage.
Or you can use this package: [@ng-web-apis/common](https://www.npmjs.com/package/@ng-web-apis/common)